docs/deployment/index.md

```diff
@@ -17,14 +17,23 @@ _For more information about advanced configuration, read our documentation [here
 ```
 
 # Kubernetes
-We currently provide [a sample declaration](https://github.com/tomkerkhove/promitor/tree/master/deploy) which deploys all the required infrastructure on your Kubernetes cluster.
+We currently provide [a helm chart](https://github.com/tomkerkhove/promitor/tree/master/charts) which deploys all the required infrastructure on your Kubernetes cluster.
 
-Once downloaded, you can deploy it by running this command:
+To use this, you will need to provide parameters [via `--set` or `--values`](https://helm.sh/docs/using_helm/#customizing-the-chart-before-installing). [Our example yaml](https://github.com/tomkerkhove/promitor/blob/master/charts/local-values.yaml.example) shows a sample configuration file.
+
+You will need to provide the following values to minimally run the chart:
+- Application Id
+- Application Key
+- Tenant Id
+- Subscription Id
+- Resource Group
+- Metric Configuration
+
+If using a values file modeled on the sample yaml, you can then deploy the chart by running this command:
 ```
-❯ kubectl apply --file .\deploy\kubernetes-spec.yaml --namespace promitor
+❯ helm install --name promitor .\charts\promitor --values \charts\local-values.yaml
```

Comment:
Can you please add a sample on how to install the helm chart with only `--set` and without passing a values file?

Reply:
Isn't really reasonable as most of the metric declaration yaml would need to be passed in